### DISCUSSION TITLE
studio visual

### DISCUSSION THREAD

**Discussion by dinaf39** [teamalgoritma/community#666] [Q&A] [answered]
![image](https://github.com/teamalgoritma/community/assets/130343206/b5a97dc6-39c9-401c-bd35-9c5b57785f72)
jika pada studio visual pada terminal terdapat keterangan ini conda : The term 'conda' is not recognized as the name of a cmdlet, function, script file, or operable program kenapa ya?

**Comment by nadyadtm** [comment, marked as answer]
Dear Bu @dinaf39 

Silakan untuk menggantinya menjadi cmd terlebih dahulu dengan menggunakan command `cmd`. Lalu silakan untuk coba aktivasi lagi environmentnya. Jika masih belum bisa, ibu dapat mengikuti step yang terdapat di diskusi ini https://github.com/teamalgoritma/community/discussions/660

Semoga membantu

**Comment by nadyadtm** [comment]
Halo @dinaf39, untuk menjaga agar forum diskusi berjalan kondusif, karena tidak ada respon lanjutan setelah lebih dari 7 hari dari respon terakhir, kami izin menandai pertanyaan sudah terjawab dan menutup diskusi ini. Semoga dapat dipahami dan terima kasih.
